src/main/java/ca/bc/gov/backendstartapi/entity/ParentTreeOrchard.java

```diff
@@ -0,0 +1,25 @@
+package ca.bc.gov.backendstartapi.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * This class represents a relation between {@link ParentTree} and {@link Orchard} in the database.
+ */
+@Getter
+@Setter
+@Entity
+@Table(name = "PARENT_TREE_ORCHARD")
+public class ParentTreeOrchard {
+
+  @Id
+  @Column(name = "PARENT_TREE_ID")
+  private Long parentTreeId;
+
+  @Column(name = "ORCHARD_ID")
+  private String orchardId;
```

Comment:
This entity has a composed primary key (checked on the test database), with both `PARENT_TREE_ID` and `ORCHARD_ID` being needed to uniquely identify a registry. Even if `ParentTreeOrchardRepository.findById()` works with this mapping, I think we'd be misleading future developers.

Reply:
Yeah. Good catch! Updates here now. Thanks